### PR TITLE
Github action

### PR DESCRIPTION
**Pull request by Kilo59**
## TODO
- [x] Add API key secret
- [x] Fix tests
- [ ] verify coveralls step
- [ ] add pypy to test matrix
- [x] remove travis file

### CODE REVIEW

**Comment by Kilo59** [comment]
To skip failing tests

https://docs.pytest.org/en/6.2.x/skipping.html#xfail-mark-test-functions-as-expected-to-fail

**Comment by ShariqT** [comment]
The coveralls step will take a little more investigation. The Coverall Github Action only takes lcov.info files, but `coverage` doesn't output anything but json, html or xml. I tried some of the suggestions found in this [issue](https://github.com/coverallsapp/github-action/issues/30). In the meantime, we can at least have the tests running and circle back in the new year about what to do with coveralls. In the thread there were people suggesting other services that will parse json and xml, so that is something else to consider.

**Review by mwj8410 — APPROVED** [review]

File: Pipfile [hunk 1]
```diff
@@ -8,14 +8,15 @@ requests = "*"
...
-coveralls = "*"
+coverage = "5.5"
+coveralls = "2.2.0"
 bumpversion = "*"
 setuptools = "*"
 twine = "*"
-pytest = "*"
-pytest-cov = "*"
+pytest = "6.2.1"
+pytest-cov = "2.11.1"
```
Comment (Kilo59) [inline Pipfile:17]: We shouldn't need to pin the package versions like this in the Pipfile.

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Pipenv Coveralls Test
+
+on: ["push", "pull_request"]
```
Comment (BennyKitchell) [inline .github/workflows/build.yaml:3]: Is this push intentional?

Comment (Kilo59) [reply .github/workflows/build.yaml:3]: It's a public repo so I don't think the minutes eaten up by it cost Lob anything.

File: Pipfile (hunk 1 above)
Comment (ShariqT) [reply Pipfile]: This has been changed to not use exact versions.

**Pull request merged**